### PR TITLE
Fix test

### PR DESCRIPTION
**Pull request by floitschG**

### CODE REVIEW

File: dev/devicelab/bin/tasks/gradle_plugin_test.dart [hunk 1]
```diff
@@ -12,88 +12,125 @@ import 'package:flutter_devicelab/framework/utils.dart';
...
+        section('gradlew assembleFreeDebug (product flavor)');
+        await project.addProductFlavor('free');
+        await project.runGradleTask('assembleFreeDebug');
+
+        await project.introduceError();
+      });
+
+      await runTest((project, pluginProject) async {
+        section('gradlew on build script with error');
+        {
```
Comment (mravn-google) [inline dev/devicelab/bin/tasks/gradle_plugin_test.dart]: This extra scoping is not needed anymore.

File: dev/devicelab/bin/tasks/gradle_plugin_test.dart [hunk 2]
```diff
@@ -12,88 +12,125 @@ import 'package:flutter_devicelab/framework/utils.dart';
...
+          if (!output.contains('Build failed') || !output.contains('builTypes'))
+            return _failure(
+                'Gradle output should contain a readable error message',
+                result);
+        }
+      });
+
+      await runTest((project, pluginProject) async {
+        section('flutter build apk on build script with error');
+        {
```
Comment (mravn-google) [inline dev/devicelab/bin/tasks/gradle_plugin_test.dart]: Same here.

**Review by mravn-google — APPROVED** [review]
LGTM

File: dev/devicelab/bin/tasks/gradle_plugin_test.dart (hunk 1 above)
Comment (floitschG) [reply dev/devicelab/bin/tasks/gradle_plugin_test.dart]: done.

File: dev/devicelab/bin/tasks/gradle_plugin_test.dart (hunk 2 above)
Comment (floitschG) [reply dev/devicelab/bin/tasks/gradle_plugin_test.dart]: done.

**Pull request merged**